metrics/src/main/java/mobile/aerogear/org/metrics/MetricsService.java

```diff
@@ -0,0 +1,112 @@
+package mobile.aerogear.org.metrics;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.pm.PackageManager;
+import android.util.Log;
+
+import org.aerogear.mobile.core.MobileCore;
+import org.aerogear.mobile.core.ServiceModule;
+import org.aerogear.mobile.core.configuration.ServiceConfiguration;
+import org.aerogear.mobile.core.http.HttpRequest;
+import org.aerogear.mobile.core.http.HttpServiceModule;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.UUID;
+
+public class MetricsService implements ServiceModule {
+    public final static String LOG_TAG = "AEROGEAR/METRICS";
+    private final static String STORAGE_NAME = "aerogear-metrics";
```

Comment:
I would normally suggest using the full package name to avoid potential naming conflicts.